include/spi.h

```diff
@@ -89,24 +89,24 @@ extern "C" {
  * Some controllers support dual, quad or octal MISO lines connected to slaves.
  * Default is single, which is the case most of the time.
  */
-#define SPI_LINES_SINGLE	(0 << 11)
-#define SPI_LINES_DUAL		(1 << 11)
-#define SPI_LINES_QUAD		(2 << 11)
-#define SPI_LINES_OCTAL		(3 << 11)
+#define SPI_LINES_SINGLE        (0 << 11)
+#define SPI_LINES_DUAL          (1 << 11)
+#define SPI_LINES_QUAD          (2 << 11)
+#define SPI_LINES_OCTAL         (3 << 11)
 
-#define SPI_LINES_MASK		(0x3 << 11)
+#define SPI_LINES_MASK          (0x3 << 11)
```

Comment:
Ditto

Reply:
Doh! These are changes that uncrustify added. New version committed with those taken out. OK, I think we're about there now and I'm better educated as a fringe benefit.  It might be an idea to a note at https://docs.zephyrproject.org/latest/contribute/index.html to the section on uncrustify for newbies like me, suggesting to _not_ use it on existing code (or at least, make it a separate commit).

Reply:
Indeed, this is much better !